message-index/messages/GHC-53786/case-expr-in-case-pattern/after/CaseExprInCasePattern.hs

```diff
@@ -0,0 +1,5 @@
+module CaseExprInFunctionDef where
+
+f x y = case x of
+  _ -> case y of
+    _ -> 1
```

Comment:
I think we should use good practice and put some kind of top-level signature on this, even if it's kind of moot
```suggestion
f :: a -> b -> Int
f x y = case x of
  _ -> case y of
    _ -> 1
```